features/renders_govspeak_editor_no_js.feature

```diff
@@ -6,8 +6,15 @@ Feature: Renders govspeak editor if JS is disabled
 
   Scenario: I create a new publication
     When I start creating a new publication
-    Then I should see the textarea instead of the visual editor
+    Then I should only see the textarea for the body field
+
+  Scenario: I edit a publication saved with visual editor
+    When I edit a publication saved with visual editor
+    Then I should only see the textarea for the body field
```

Comment:
Should we say "govspeak editor" instead of "textarea" here (and elsewhere in these tests) for clarity, or is that two different things?

Reply:
Subtle differences left, right and centre.

If no JS -> then it is truly only the textarea, since the preview button is also not shown. This one makes some sense but yes the name of the step is a bit too technical. Let me think of an alternative.


The more concerning naming yet! is where we have gospeak editor (original one) vs govspeak editor (rendered from the visual editor template). Really nothing in it to the user but I just could not have steps named the same 😅

Reply:
Hmm. Ideally cucumber tests would reflect the experience of the user, being very much a BDD framework. It is a bit of a blunt tool though so fair enough.